### ISSUE TITLE
Support for placeholder tags autocomplete

### ISSUE THREAD

**Issue by KraXen72** [wkillerud/vscode-scss#44]
i have a mappings definitions that extends classes by placeholders, see:
```scss
.lookFilled-1H2Jvj {
	@extend %buttonFilled !optional;
}
.lookOutlined-3RTC7c {
	@extend %buttonOutlined !optional;
}
.lookInverted-2GrLaB {
	@extend %buttonInverted !optional;
}
.colorRed-2VFhM4 {
	@extend %buttonColorRed !optional;
}
.colorGreen-jIPCAS {
	@extend %buttonColorGreen !optional;
}
.buttonColor-3bP3fX {
	@extend %panelActionButtonColor !optional;
}
```

now, in a different file in the same workspace, i would like to get autocompletion when starting to type for example `%buttonF`

**Comment by wkillerud** [comment]
Thanks for opening an issue. I’m working on placeholder support across the board, not just completions. Hope to have something published soon.

**Comment by wkillerud** [comment]
Published as part of [version 2.11.0](https://github.com/wkillerud/vscode-scss/releases/tag/2.11.0).

I don't use placeholder selectors in any of my current projects, so feel free to open new issues if something doesn't work the way you expect @KraXen72 :)

**Comment by KraXen72** [comment]
yoo. thanks a ton! will update you on how it's working/behaving
